copyables/entrypoint.sh

```
@@ -156,15 +156,19 @@ if [ ! -f $CONFIG ] || [ ! -s $CONFIG ]; then
   if [[ $VPNCMD_SERVER ]]; then
+    set -f
     while IFS=";" read -ra CMD; do
       vpncmd_server $CMD
     done <<<"$VPNCMD_SERVER"
+    set +f
   fi
 
   if [[ $VPNCMD_HUB ]]; then
+    set -f
     while IFS=";" read -ra CMD; do
       vpncmd_hub $CMD
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)